templates_jinja2/suggest_apiwrite.html

```diff
@@ -0,0 +1,101 @@
+{% extends "base.html" %}
+
+{% block title %}The Blue Alliance - Request Write Keys{% endblock %}
+
+{% block meta_description %}Request auth tokens to import your offseason event to The Blue Alliance{% endblock %}
```

Comment:
This is in the `<meta>` tag for the page, you sure about that? I can add some links to the overview paragraph, however